src/js/components/DataChart/stories/GuideDefault.js

```diff
@@ -16,9 +16,9 @@ for (let i = 0; i < 7; i += 1) {
 const Example = () => (
   <Grommet theme={grommet}>
     <Box align="center" justify="start" pad="large">
-      <DataChart data={data} chart={{ key: 'percent' }} xAxis yAxis />
+      <DataChart data={data} series="percent" guide />
```

Comment:
Is it expected that the last bar is getting out of bounds of the container?
![image](https://user-images.githubusercontent.com/6320236/90075427-9d576680-dcba-11ea-8c13-3d3ed8f0eae7.png)

Reply:
It is expected. All vertical guide lines are drawn at the center of the bars.